src/DevBetterWeb.Infrastructure/Handlers/MemberAddressUpdatedHandler.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+using DevBetterWeb.Core.Events;
+using DevBetterWeb.Core.Interfaces;
+using Microsoft.Extensions.Configuration;
+using Newtonsoft.Json.Linq;
+
+namespace DevBetterWeb.Infrastructure.Handlers
+{
+  class MemberAddressUpdatedHandler : IHandle<MemberAddressUpdatedEvent>
+  {
+    public MemberAddressUpdatedHandler(IConfiguration configuration)
+    {
+      _configuration = configuration;
```

Comment:
This handler should depend on a service for getting coordinates, not the config for the google key. Also the _configuration property should probably be a private field (if we were keeping it here).

What I would recommend instead is:
- Create an interface in Core/Interfaces for something like IMapCoordinateService with method `Task<string> GetMapCoordinates(string address)`
- Create a service in Infrastructure/Services called GoogleMapCoordinateService : IMapCoordinateService and put the private method you have in this class into the service. Inject IConfiguration into the service.
- Inject the `IMapCoordinateService` into this handler and just call it.
- Consider injecting an `HttpClient` into the MapCoordinateService rather than newing it up in a using block. I think this will be better but I'm not 100% sure since the guidance on this has changed back and forth a bit.

Reply:
This is done. 
Thank you for giving me the steps.